mlrun/serving/routers.py

```diff
@@ -29,6 +29,7 @@
 import mlrun.common.model_monitoring
 import mlrun.common.schemas.model_monitoring
 from mlrun.utils import logger, now_date
+import mlrun.feature_store as fs
```

Comment:
```suggestion
import mlrun.feature_store as fstore
```
fs can mean feature set or feature store, and for this reason we changed it everywhere to disambiguate.